helios-client/src/main/java/com/spotify/helios/common/descriptors/ServicePortParameters.java

```diff
@@ -21,27 +21,56 @@
 
 package com.spotify.helios.common.descriptors;
 
-import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.Objects;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.List;
 
 /**
  * A class that used to do something, but now is just effectively a placeholder in a Map
```

Comment:
Perhaps update this comment to be like [this](https://github.com/spotify/helios/blob/master/helios-client/src/main/java/com/spotify/helios/common/descriptors/Job.java#L47)?